Update Townie docs (#209)
val-town/val-town-docs: src/content/docs/reference/townie.mdx

Once Townie has created a val, you'll see a preview of the live code.
You can continue the conversation to iterate on your val further or make manual edits to the code if needed.

## HTTP val Template

Townie's default system prompt includes a template that is used as a basis for vals created in Townie. The template is intended to work well for code that can run on the server or in a client-side context. By default, it'll tend to use React to build front-end interfaces

> and the built-in [SQLite][/std/sqlite/] storage for persisting data.

Reviewer: ```suggestion
and the built-in [SQLite][] storage for persisting data.
```

### Diffs

> You can ask Townie "to make changes as unified diffs." For small changes, this works well, but can be error prone on larger diffs or when Townie hallucinates.

Reviewer: ```suggestion
You can ask Townie to "respond with a diff." For small changes, this works well, but can be error prone on larger diffs or when Townie hallucinates.
```
The specific language doesn't technically matter here so much, but I think this sounds better
